visualization/dashboard/src/MLIDashboard/Controls/GlobalExplanationTab/GlobalExplanationTab.tsx

```diff
@@ -166,13 +153,53 @@ export class GlobalExplanationTab extends React.PureComponent<IGlobalExplanation
                     jointDataset={this.props.jointDataset}
                     yAxisLabels={[localization.GlobalTab.aggregateFeatureImportance]}
                     sortArray={this.state.sortArray}
+                    chartType={this.state.chartType}
                     startingK={this.state.startingK}
                     unsortedX={this.props.metadata.featureNamesAbridged}
                     unsortedSeries={this.activeSeries}
-                    topK={this.props.globalBarSettings.topK}
+                    topK={this.state.topK}
                     onFeatureSelection={this.handleFeatureSelection}
                     selectedFeatureIndex={this.state.selectedFeatureIndex}
                 />
+                <IconButton
+                    className={classNames.chartEditorButton}
+                    onClick={this.toggleCalloutOpen} 
+                    iconProps={{iconName: "AreaChart"}}
+                    id={this._chartConfigId}/>
+                {(this.state.calloutVisible) && <Callout
+                    className={classNames.callout}
+                    gapSpace={0}
```

Comment:
Ohh interesting - I think the default value for gapSpace is 0. Does it affect the position of the callout when you give gapSpace=0?

Reply:
hmm, I may have been playing with different values, and settled on 0? this is just movign the existing code, so who knows why that was written that way...